apps/web/app/_utils.tsx

```diff
@@ -103,3 +104,22 @@ export const _generateMetadata = async (
     },
   };
 };
+
+export const generateMeetingMetadata = async (
```

Comment:
The key difference is `const image = SEO_IMG_OGIMG + constructMeetingImage(meeting);`. Using constructMeetingImage is important.